sru-scripts/manual/gce-sru

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+# Manual GCE upgrade and clean install validation Xenial, Bionic, Cosmic
+set -ex
+SRU_VARS=./sru-scripts/sru-vars.template
+
+if [ ! -f $SRU_VARS ]; then
+   echo "Missing config environment file $SRU_VARS."
+   exit 1
+fi
+
+. $SRU_VARS
+
+parse_args $0 $@
+assert_expected_vars PROPOSED_SCRIPT SAMPLE_CLOUDCONFIG SSHOPTS GCE_ZONE \
+    NETCFG_FILE
```

Comment:
Should `NETCFG_FILE` also be added to `sru-vars.template` ? Also, should we have different name for it when using on GCE and Oracle ?

Reply:
Yes this NETCFG_FILE is already moved to sru-vars.template in PR #127  So once both land, we'll have this available properly.